package.json

```diff
@@ -8,6 +8,8 @@
   "dependencies": {
     "bootstrap": "^3.3.4",
     "crypto-js": "^3.1.2-5",
+    "es5-shim": "^4.1.7",
```

Comment:
It looks like the ES6 shim is not necessary because we're using `babel-loader`. I'm taking out `es5-shim` and `es6-shim`. Tested in [Firefox 12 which doesn't support](https://developer.mozilla.org/en-US/docs/Web/JavaScript/New_in_JavaScript/ECMAScript_6_support_in_Mozilla) `Set()` and `Map()`.